src/rt/util/container/treap.d

```diff
@@ -32,7 +32,7 @@ nothrow:
         rand48.defaultSeed();
     }
 
-    void insert(E element)
+    @nogc void insert(E element)
```

Comment:
the @nogc in treap propagates very far down the GC due to the opApply delegates and I'm not very familiar with it at all.